### PR TITLE
merge nftp into naomq_cli

### PR DESCRIPTION
**Pull request by OdyWayne**

### CODE REVIEW

**Comment by OdyWayne** [comment]
further polish need to be done

File: nanomq_cli/include/main.h [hunk 1]
```diff
@@ -9,5 +9,6 @@
 #include "vsomeip_gateway.h"
 #include "rule_ctrl.h"
 #include "dds_client.h"
+#include "nftp _client.h"
```
Comment (wanghaEMQ) [inline nanomq_cli/include/main.h:12]: An typo????

File: nanomq_cli/include/nftp _client.h [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Author: wangha <wangha at emqx dot io>
```
Comment (wanghaEMQ) [inline nanomq_cli/include/nftp _client.h:1]: Filename ????

Comment (OdyWayne) [reply nanomq_cli/include/nftp _client.h:1]: most codes are originally from wanghamax, but I will modify that. This PR is still wip, so chill. XD

Comment (wanghaEMQ) [reply nanomq_cli/include/nftp _client.h:1]: Fine. I have turned it to a draft pr

File: nanomq_cli/include/main.h (hunk 1 above)
Comment (OdyWayne) [reply nanomq_cli/include/main.h:12]: what's wrong with that? That's not a typo.

**Comment by OdyWayne** [comment]
Logs from libnftp is still disturbing.

File: nanomq_cli/include/main.h (hunk 1 above)
Comment (OdyWayne) [reply nanomq_cli/include/main.h]: That is a typo, my bad.

**Comment by OdyWayne** [comment]
this can be merge now

**Review by wanghaEMQ — APPROVED** [review]

**Pull request merged**